Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml

```diff
@@ -2967,6 +2963,14 @@ if (propertiesFile.exists()) {
           <con:name>ActiveParticipant.@UserID</con:name>
           <con:value>kskagerb</con:value>
         </con:property>
+        <con:property>
+          <con:name>ActiveParticipant_Dest_g0.@UserID</con:name>
+          <con:value>https://localhost:8181/Gateway/DocumentSubmission/1_1/DocumentRepositoryXDR_Service</con:value>
+        </con:property>
+        <con:property>
+          <con:name>ActiveParticipant_Dest_g1.@UserID</con:name>
+          <con:value>https://localhost:8181/Gateway/DocumentSubmission/2_0/DocumentRepositoryXDR_Service</con:value>
```

Comment:
Right now properties are loaded at each testcase level. We have individual testcases per Service. In my opinion, Identifying common properties and moving them to appropriate suite / project level is out of this ticket scope and it will be addressed as part of FHAC-609.